### PR TITLE
docs: add committers promoted on 10/24/2024

### PR DESCRIPTION
**Pull request by ingomueller-net**
This PR adds the committers promoted by the [vote](https://groups.google.com/g/substrait/c/abti2mcJhH8/m/wVRuSdwSAgAJ) on October 24, 2024 to the relevant document on the web site.

### CODE REVIEW

**Comment by ingomueller-net** [comment]
@blizzara, @brunovolpato, @bvolpato, @anshuldata: @scgkiran: Please verify your names and affiliations. The current values are based on my best effort using information from Github, LinkedIn, and web search.

File: site/data/committers.yaml [hunk 1]
```diff
@@ -28,3 +28,13 @@
   Association: Voltron Data
 - Name: Matt Topol
   Association: Voltron Data
+- Name: Ingo Müller
+  Association: Google
+- Name: "@blizzara"
+  Association: unknown
```
Comment (Blizzara) [inline site/data/committers.yaml:34]:
```suggestion
- Name: Arttu Voutilainen
  Association: Palantir Technologies
```

Thanks! :)

**Review by bvolpato — DISMISSED** [review]
Thank you!

**Review by jacques-n — APPROVED** [review]

**Review by bvolpato — APPROVED** [review]

**Review by EpsilonPrime — APPROVED** [review]
Seems like we're waiting for a few more checks and then we can merge.

**Comment by ingomueller-net** [comment]
@anshuldata: can you confirm your name and affiliation?

**Comment by anshuldata** [comment]
> @anshuldata: can you confirm your name and affiliation?

Details are correct. Thanks

**Comment by anshuldata** [comment]
Approved pull request. Thanks

**Comment by ingomueller-net** [comment]
I think everyone approved:

- @ingomueller-net: PR author
- @Blizzara: proposed [change](https://github.com/substrait-io/substrait/pull/752#discussion_r1863799295)
- @bvolpato: 👍, PR, approved, [comment](https://github.com/substrait-io/substrait/pull/752#pullrequestreview-2470397069)
- @scgkiran: 👍 

Github still requires (a subset of) @cpcloud, @vbarua, and @westonpace to approve the PR.

**Review by mbrobbel — APPROVED** [review]

**Pull request merged**